### PR TITLE
IO LTC6813 Restructure

### PR DESCRIPTION
**Pull request by DJ90864**
moved the cell voltage structure to Io_LTC6813CellVoltages.c, tested on the Car and it works.

### Summary

BMS LTC6813 IO Driver Refactor done and tested on car

### Changelist 

added function in the IO Driver for LTC6813
modified Accumulator Struct with new function to get Voltage
modified BMS tests to work with the new function structure
modified CAN Message names to work with MacCAN

### Checklist
*Please change `[ ]` to `[x]` when you are ready.*
- [x] I have read and followed the code conventions detailed in [README.md](../README.md) (*This will save time for both you and the reviewer!*).
- [ ] If this pull request is longer then **500** lines, I have provided *explicit* justification in the summary above explaining why I *cannot* break this up into multiple pull requests (*Small PR's are faster and less painful for everyone involved!*).

### CODE REVIEW

**Comment by DJ90864** [comment]
BMS LTC6813 IO Driver Refactor done and tested on car

File: firmware/thruna/BMS/Test/Src/Test_Faults.cpp [hunk 1]
```diff
@@ -173,9 +145,11 @@ class BmsFaultTest : public BaseStateMachineTest
         RESET_FAKE(read_page);
         RESET_FAKE(write_page);
         RESET_FAKE(page_erase);
+        RESET_FAKE(get_cell_voltage);
 
         // Set initial voltages to nominal value
-        set_all_cell_voltages(3.8);
+        // set_all_cell_voltages(3.8);
```
Comment (will-chaba) [inline firmware/thruna/BMS/Test/Src/Test_Faults.cpp:151]: I think you meant to delete this line

File: scripts/code_generation/jsoncan/src/codegen/dbc_generation/dbc_generation.py [hunk 2]
```diff
@@ -17,10 +17,6 @@
 
 {attribute_definitions}
 
-{start_value_attributes}
-
-{cycle_time_attributes}
-
```
Comment (will-chaba) [inline scripts/code_generation/jsoncan/src/codegen/dbc_generation/dbc_generation.py:23]: @gtaharaedmonds I'm assuming this has to do with Mac CAN viewing?

Comment (will-chaba) [inline scripts/code_generation/jsoncan/src/codegen/dbc_generation/dbc_generation.py:23]: @gtaharaedmonds I'm guessing this has to do with Mac CAN viewing?

File: firmware/thruna/BMS/Test/Src/Test_Faults.cpp (hunk 1 above)
Comment (will-chaba) [inline firmware/thruna/BMS/Test/Src/Test_Faults.cpp:151]: I think you meant to delete this line
```suggestion
```

**Review by will-chaba — APPROVED** [review]
Looks good to me!

File: scripts/code_generation/jsoncan/src/codegen/dbc_generation/dbc_generation.py (hunk 2 above)
Comment (gtaharaedmonds) [reply scripts/code_generation/jsoncan/src/codegen/dbc_generation/dbc_generation.py:23]: Yep, `cantools` doesn't like the attributes for some reason. We probably should've dropped a TODO or something but this is OK

**Pull request merged**